tools/wt_cmp_dir

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# Compare content in WT home directories.
+
+usage_exit() {
+    echo "Usage: wt_cmp_dir [ -vq ] [ -t timestamp ] dir1 [ -t timestamp  ] dir2"
+    echo "   -v   verbose"
+    exit 1
+}
+
+wtfiles() {
+    local tmpa=/tmp/wcd$$a
+    local tmpb=/tmp/wcd$$b
+    lastwt=
+    wt=wt
+    while [ "$wt" != "$lastwt" -a ! -x "$wt" ]; do
+        wt=../$wt
+    done
+    #TODO: check for corruption
+    $wt -h "$1" list >$tmpa 2>$tmpb
+    if [ "$?" != 0 -o -s $tmpb ]; then
+        echo failed command: $wt -h "$1" list
+        cat $tmpb
+        rm -f $tmpa $tmpb
+        exit 1
+    fi
+    uris=$(grep -v colgroup: < $tmpa)
+    rm -f $tmpa $tmpb
+    #echo INITIAL URIS $uris >&2
+    results=""
+    for uri in $uris; do
+        case "$uri" in
+            file:* )
+                base=$(echo "$uri" | sed -e 's/file://' -e 's/.wt//')
```

Comment:
The `sed` command would be cleaner if it specified that `file` should be at the beginning of the string and `.wt` at the end, i.e.: `sed -e 's/^file://' -e 's/.wt$//'`